git/git.go

```diff
@@ -121,12 +122,15 @@ func loadFilePathsList(opts *options.Options, provider *repositoryProvider) erro
 			return fmt.Errorf("failed to read paths file from location: '%v', error: '%v'", opts.PathsFileLocation, err)
 		}
 
+		reader := csv.NewReader(file)
 		defer file.Close()
-		scanner := bufio.NewScanner(file)
 
-		for scanner.Scan() {
-			line := scanner.Text()
-			provider.fileListToSnap[line] = true
+		lines, err := reader.ReadAll()
+		if err != nil {
+			return fmt.Errorf("failed to read paths file from location: '%v', error: '%v'", opts.PathsFileLocation, err)
+		}
+		for i := range lines {
+			provider.fileListToSnap[lines[i][0]] = true
```

Comment:
Do you think we need some validation that `lines[i]` isn't empty? or do we trust the flow?

Reply:
So I am writing this as a CSV from C#, and I didn't see any way that this broke when I tested it...
Do you have a specific case in mind you want me to test?